docs/getting-started/dockstore-workflows.rst

```diff
@@ -46,9 +47,13 @@ below:
 |                        | - A workflow section that runs the task  | - A workflow section that connects the tasks  |
 |                        | - An associated Docker image             |                                               |
 +------------------------+------------------------------------------+-----------------------------------------------+
-| Nextflow               | - N/A                                    | - Any valid nextflow workflow                 |
-|                        |                                          |                                               |
+| Nextflow               | - N/A                                    | - Any valid Nextflow workflow                 |
 +------------------------+------------------------------------------+-----------------------------------------------+
+| Galaxy                 | - N/A*                                   | - Any valid Galaxy workflow                   |
++------------------------+------------------------------------------+-----------------------------------------------+
+
+\* There are tools that make up Galaxy workflows from the Galaxy toolbox or ToolShed.
+Dockstore does not support registration of these yet.
```

Comment:
Will we ever support them? I don't understand them very well, but my limited understanding is that they are different enough that we would never would. If we never would, then I would remove the word `yet`. Otherwise keep it in.